src/components/BottomBar/Controls.js

```diff
@@ -131,40 +133,108 @@ export const Controls = controlsInjector(observer(({ store, history, annotation
 
     const look = (disabled || submitDisabled) ? 'disabled' : 'primary';
 
-    if ((userGenerate && !sentUserGenerate) || (store.explore && !userGenerate && store.hasInterface('submit'))) {
-      const title = submitDisabled
-        ? 'Empty annotations denied in this project'
-        : 'Save results: [ Ctrl+Enter ]';
-      // span is to display tooltip for disabled button
+    const SubmitOption = ({ isUpdate }) => {
```

Comment:
This Button is only used in the FF'd case, lets put the definition within the FF as well to ensure that is clear.

Reply:
good thinking